mono/sgen/sgen-gc.c

```diff
@@ -1802,6 +1802,12 @@ collect_nursery (const char *reason, gboolean is_overflow)
 	TV_GETTIME (btv);
 	time_minor_scan_roots += TV_ELAPSED (atv, btv);
 
+	SGEN_LOG (2, "Minor scan roots: %lld usecs (accumulated major scan=%lld, accumulated los scan=%lld, scan jobs=%d)",
+		(long long)(TV_ELAPSED (atv, btv) / 10),
+		(long long)((time_minor_scan_major_blocks - major_scan_start) / 10),
+		(long long)((time_minor_scan_los - los_scan_start) / 10),
+		sgen_workers_get_job_split_count (GENERATION_NURSERY));
```

Comment:
What kind of information the split count would provide to the users ? I think it would make more sense to log the number of workers instead, indicating how ideal is the work distribution.

Reply:
When I added that information it gave some valuable info around how many jobs that accumulated into the respective major/los times, so you could make an average, but we could of course use the number of workers instead, that will give us the average scan time per worker.

Reply:
You where thinking around using this worker_contexts [generation].active_workers_num instead of sgen_workers_get_job_split_count? If so we will need to add new function to get that info for sgen workers so it could be used in this logging.

Reply:
feel free to add a method to get that number